### PR TITLE
SAN 2968 log rabbit error

### PR DESCRIPTION
**Pull request by podviaznikov**
1. Log error error to rollbar
2. Cleanup some code
3. Rabbit client is more unit testable now

### CODE REVIEW

**Comment by podviaznikov** [comment]
Todo:
- [ ] pass code review
- [ ] stage on beta or gamma

**Comment by podviaznikov** [comment]
@bkendall, @anandkumarpatel please review this small PR when you have time

File: test/events.js [hunk 1]
```diff
@@ -105,11 +105,11 @@ describe('events#enhance', function () {
         });
     });
     ['random', 'anton', 'anand', 'untag', 'delete']
-      .forEach(function(event) {
-        it('should NOT add inspect data if other event:'+event, function (done) {
+      .forEach(function(ev) {
```
Comment (anandkumarpatel) [inline test/events.js:108]: why did you make this less descriptive?

**Comment by anandkumarpatel** [comment]
code LGTM, need to test.

File: test/events.js (hunk 1 above)
Comment (podviaznikov) [inline test/events.js:108]: because `event` is highlighted by a lot of editors and github as JS keyword and it isn't

File: test/rabbitmq.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  describe('on error', function () {
+    it('should call _handleFatalError', function (done) {
+      sinon.stub(Hermes.prototype, 'connect').yields(null);
+      rabbitmq.connect(function (err) {
+        if (err) {
+          return done(err);
+        }
+        expect(function () {
+         rabbitmq.rabbit.emit('error');
+       }).to.throw();
```
Comment (bkendall) [inline test/rabbitmq.js]: tabbing is off here

**Comment by bkendall** [comment]
looks just peachy to me

**Comment by podviaznikov** [comment]
@anandkumarpatel and I verified beta

**Pull request merged**